src/emails/templates/org-invite.tsx

```
@@ -0,0 +1,154 @@
+          </p>
+        </Text>
+      </EmailLayout>
+    );
+  }
+};
+
+export const getTemplate: GetTemplate = async (props) => {
+  return await render(<Template {...props} />, { plainText: props.plainText });
+};
+
+export const getSubject: GetSubject = async (_props) => {
```

ESLint

[error] '_props' is defined but never used.

(@typescript-eslint/no-unused-vars)